pkgs/applications/audio/lv2lint/default.nix

```
@@ -0,0 +1,21 @@
+{ stdenv, lib, fetchurl, pkg-config, meson, ninja, lv2, lilv, curl, libelf }:
+
+stdenv.mkDerivation rec {
+  pname = "lv2lint";
+  version = "0.14.0";
+
+  src = fetchurl {
+    url = "https://git.open-music-kontrollers.ch/lv2/${pname}/snapshot/${pname}-${version}.tar.xz";
+    sha256 = "sha256-yPKM7RToLNBT+AXSjfxxpncESmv89/wcGCt//pnEGqI=";
+  };
+
+  nativeBuildInputs = [ pkg-config meson ninja ];
+  buildInputs = [ lv2 lilv curl libelf ];
+
+  meta = with lib; {
+    homepage    = "https://open-music-kontrollers.ch/lv2/${pname}:";
+    license     = licenses.artistic2;
```

```suggestion
    homepage = "https://open-music-kontrollers.ch/lv2/${pname}:";
    license = licenses.artistic2;
```
What are the extra spaces for?

---

Misguided attempt at alignment?